packages/connect/src/api/firmware/getBinaryForFirmwareUpgrade.ts

```diff
@@ -63,5 +63,5 @@ export const getBinaryForFirmwareUpgrade = ({
         );
     }
 
-    return getBinary({ releases, baseUrl, version, btcOnly, intermediaryVersion });
```

Comment:
intermediary already consumed on line 33 in this file no need to pass down to `getBinary`